views/layout.haml

```diff
@@ -1,6 +1,7 @@
 !!!
 %html
   %head
+    %link{rel: 'stylesheet', href: "#{ENV['CDN_URL']}/fonts/stylesheet.css"}
```

Comment:
Please document this env var somewhere. Either in the readme or in a separate file with available variables.

Reply:
Fixed